### PR TITLE
Add [AUR] maintainer and last modified badges

### PR DESCRIPTION
**Pull request by mrf345**
Hello there! I'm adding two additional badges to the `AUR` service, [_related issue_]( https://github.com/badges/shields/issues/4656):

- image

-  image

Thanks for the awesome service!

### CODE REVIEW

**Comment by mrf345** [comment]
I guess that's a flaky [test result](https://circleci.com/gh/badges/shields/99008?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) 🤔  it's passing locally:

image

**Comment by PyvesB** [comment]
Thanks for you contribution @mrf345 ! Let's see if re-running the tests makes them pass. 😉

**Comment by mrf345** [comment]
Hi @PyvesB , my pleasure! I don't have write access to rerun the tests

image

so i'm just changing the ref and force pushing with `git commit --amend && git push -f` I tried it couple of times yesterday and just now but same [failing tests](https://circleci.com/gh/badges/shields/99027?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link). yet the tests locally are passing 🤔

image

**Comment by PyvesB** [comment]
Re-running the test suite for this PR did not help. That test did not fail in [last daily CI run](https://app.circleci.com/pipelines/github/badges/daily-tests/173/workflows/a0b8d1c9-17bb-4a66-893f-376ec1453b04/jobs/1068/tests) though. Maybe the API response for `gog-gemini-rue` is missing a `Maintainer` or `LastModified` field, which you've added to the AUR schema in this PR? Could you please check that?

**Comment by mrf345** [comment]
Thanks man. You're 100% on point! turns out i was missing `.allow(null)` in `Maintainer` schema.
Just managed to reproduce it locally after running `npx mocha --delay core/service-test-runner/cli.js --only=aur`:

image

should be passing soon 🤞

File: services/aur/aur.service.js [hunk 1]
```diff
@@ -168,8 +174,85 @@ class AurVersion extends BaseAurService {
...
+    return { label: 'maintainer' }
+  }
+
+  static render({ maintainer }) {
+    return { message: maintainer, color: 'blue' }
+  }
+
+  async handle({ packageName }) {
+    const json = await this.fetch({ packageName })
+    return this.constructor.render({ maintainer: json.results.Maintainer })
```
Comment (calebcartwright) [inline services/aur/aur.service.js:209]: Given that we had to update the schema to account for the `Maintainer` key having a null value, we'll need to check for that scenario and handle it accordingly. Something like the below could work (obligatory disclaimer about potential prettier/eslint/etc. issues with inline code typed in the GitHub UI), note you'd need to also add the import `InvalidResponse` from `..` up on line 7

```suggestion
    const { results: { Maintainer: maintainer } } = await this.fetch({ packageName })
    if (!maintainer) {
      throw new InvalidResponse({ prettyMessage: 'No maintainer' })
    }
    return this.constructor.render({ maintainer })
```

File: services/aur/aur.service.js [hunk 2]
```diff
@@ -168,8 +174,85 @@ class AurVersion extends BaseAurService {
...
+
+  static render({ date }) {
+    const color = ageColor(date)
+    const message = formatDate(date)
+    return { color, message }
+  }
+
+  async handle({ packageName }) {
+    const json = await this.fetch({ packageName })
+    const date = 1000 * parseInt(json.results.LastModified)
```
Comment (calebcartwright) [inline services/aur/aur.service.js:247]: I don't suppose there's a spec/doc for this API anywhere is there? If so might be helpful to include that link in an inline comment for future eyes

File: services/aur/aur.service.js [hunk 3]
```diff
@@ -168,8 +174,85 @@ class AurVersion extends BaseAurService {
   }
 }
 
+class AurMaintainer extends BaseAurService {
+  static get category() {
+    return 'social'
```
Comment (calebcartwright) [inline services/aur/aur.service.js:179]: The category for this one is a little tricky, but I'm thinking we should use a different one (maybe `other`). The social badges are a different breed and almost always have a very unique style (https://shields.io/category/social).

This is more of just an additional attribute badge for a package

File: services/aur/aur.service.js (hunk 1 above)
Comment (mrf345) [reply services/aur/aur.service.js:209]: Thanks for the great insight and example! I've missed that fail-safe. although, i thought there's some sort of generic error handle that catch them.

File: services/aur/aur.service.js (hunk 2 above)
Comment (mrf345) [reply services/aur/aur.service.js:250]: There's am arch [wiki page](https://wiki.archlinux.org/index.php/Aurweb_RPC_interface) it has a reference of the expected parameters. But no details regarding data-types.

File: services/aur/aur.service.js (hunk 3 above)
Comment (mrf345) [reply services/aur/aur.service.js:179]: Good call! `others` is definitely a better fit.

**Pull request merged**